common-libs/common/src/Affinity.ts

```diff
@@ -274,6 +322,17 @@ export class Affinity {
         controller: await didDocumentService.buildDidDocument(),
       }),
     })
+
+    // send VC_SIGNED event
+    const isTestEnvironment = process.env.NODE_ENV === 'test'
```

Comment:
i only found this under common-libs dir:

```
❯ git grep -w process.env
common-libs/common/src/Affinity.ts:      const isTestEnvironment = process.env.NODE_ENV === 'test'
common-libs/common/src/Affinity.ts:    const isTestEnvironment = process.env.NODE_ENV === 'test'
common-libs/common/src/shared/profiler/DefaultProfilerActivator.ts:  if (process && process.env) {
common-libs/common/src/shared/profiler/DefaultProfilerActivator.ts:    return Object.keys(process.env).findIndex((i) => profilerVars.includes(i)) !== -1
```

First two lines are mine. Not sure the effect of the profiler though.
So if we want to avoid sending metrics when it is running test, what will be the go-to approach here at common-lib?